akka-persistence/src/test/scala/akka/persistence/EndToEndEventAdapterSpec.scala

```diff
@@ -228,7 +228,7 @@ abstract class EndToEndEventAdapterSpec(journalName: String, journalConfig: Conf
         .withoutPath(s"""$journalPath.event-adapter-bindings."${classOf[EndToEndEventAdapterSpec].getCanonicalName}$$A"""")
 
       withActorSystem("MissingAdapterSystem", journalConfig.withFallback(missingAdapterConfig)) { implicit system2 ⇒
-        EventFilter[ActorInitializationException](occurrences = 1, pattern = ".*undefined event-adapter.*") intercept {
+        EventFilter[ActorInitializationException](occurrences = 1, pattern = ".*akka.persistence.journal.leveldb.*") intercept {
```

Comment:
LeveldbJournal constructor takes config parameter, and reflection uses Constructor.newInstance method to instantiate it.
[ Constructor.newInstance() always wraps the thrown exception with an InvocationTargetException.](https://docs.oracle.com/javase/tutorial/reflect/member/ctorInstance.html)

so ActorInitializationException message is changed